### DISCUSSION TITLE
docker-compose installation

### DISCUSSION THREAD

**Discussion by invin12** [allure-framework/allure2#1574] [TestOps]
Step by step installation from this instruction
https://docs.qameta.io/allure-testops/getstarted/docker-compose/
VERSION=3.190.2

Initialization was successful, but I can't login to the user interface with the default credentials(admin/admin)
After successful login, page reload back to empty login page

I see this error in Chrome devel console
app.vendors.afb0f8c13360c88cb2f7.bundle.js:2 GET http://10.189.219.96/api/uaa/account/me 401 (Unauthorized)

**Comment by cheshi-mantu** [comment]
@invin12 
Hi there
please consider for future cases (if any) using the help-desk for Allure TestOps support (https://support.qameta.io, use your corporate email address for registering). That'll be much faster.
for your case, if you aren't using https, please check if ALLURE_SECURE is enabled in docker-compose.yml in gateway settings. if it is and you aren't using https, then comment this parameter. then use `down` and `up -d` to apply the changes.

**Reply by invin12** [reply]
ALLURE_SECURE: "false" solved the issue
Thank you!

**Reply by cheshi-mantu** [reply]
please remember if you are going to use Jira integration, it has to be enabled, set to true and testop is to be hidden behind https (nginx for example).
